packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -101,23 +101,20 @@ class RenderAndroidView extends RenderBox with _PlatformViewGestureMixin {
   bool _isDisposed = false;
 
   /// The Android view controller for the Android view associated with this render object.
-  AndroidViewController get viewController => _viewController;
-  AndroidViewController _viewController;
+  AndroidViewController get viewController => controller as AndroidViewController;
+
   /// Sets a new Android view controller.
   ///
   /// `viewController` must not be null.
   set viewController(AndroidViewController viewController) {
-    assert(_viewController != null);
-    assert(viewController != null);
-    if (_viewController == viewController)
+    if (viewController == controller)
       return;
-    _viewController.removeOnPlatformViewCreatedListener(_onPlatformViewCreated);
-    _viewController = viewController;
+
+    (controller as AndroidViewController).removeOnPlatformViewCreatedListener(_onPlatformViewCreated);
+    controller = viewController;
```

Comment:
this calls the setter in the parent class. I don't like this API, but I'm trying to make this PR a non-breaking change.